blocks.js

```diff
@@ -7553,13 +7554,41 @@ InputSlotMorph.prototype.dropDownMenu = function (enableKeyboard) {
         if (Object.prototype.hasOwnProperty.call(choices, key)) {
             if (key[0] === '~') {
                 menu.addLine();
-            // } else if (key.indexOf('§_def') === 0) {
-            //     menu.addItem(choices[key].blockInstance(), choices[key]);
+        // } else if (key.indexOf('§_def') === 0) { 
+        // menu.addItem(choices[key].blockInstance(), choices[key]);
+            } else if (key.charCodeAt(key.length - 1) == 0x25ba) {
```

Comment:
IMO, the idea of using something not easily type-able was the correct one, since `{ }` would mess up dropdowns for variable selection, like `~` does.